Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -83,7 +87,12 @@ import BraintreeDataCollector
         _ request: BTPayPalVaultRequest,
         completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void
     ) {
-        tokenize(request: request, completion: completion)
+        // Check if PayPal app is installed
+        if request.enablePayPalAppSwitch == true {
```

Comment:
We don't want to diverge in the flow _just yet_. We still need to get the response back from[ this API call](https://github.com/braintree/braintree_ios/blob/4d4f107700b1483addeae0b3b7c741014f398a5d/Sources/BraintreePayPal/BTPayPalClient.swift#L248) which will either contain the Venice URL, or it won't. Based on if the URL present, we will or won't proceed with app switch flow.

Reply:
This is where we instead want to do the `appInstalled` check. If that check fails, we don't want to bother encoding [these additional parameters](https://github.com/braintree/braintree_ios/pull/1228)

Reply:
Thanks! I've updated the PR with the changes